lib/src/test/java/ca/jbrains/pos/test/TestScanningAnItem.java

```diff
@@ -13,18 +13,18 @@
 public class TestScanningAnItem {
     @Test
     void scanOneItem() {
-        Stream<String> lines = makeBufferedReader(List.of("12345"));
+        Stream<String> lines = makeBarcodeBufferedReader(List.of("12345"));
         Assertions.assertEquals(List.of("12345"), lines.collect(Collectors.toList()));
     }
 
-    private Stream<String> makeBufferedReader(List<String> barcodes) {
+    private Stream<String> makeBarcodeBufferedReader(List<String> barcodes) {
```

Comment:
I think, it's likely that we would need to extract this bufferedReader into some kind of interface like BarcodeReader, so might as well this the method name obvious that it's creating barcodebufferedreader

Reply:
I like the extra precision, but then we have duplication between "make barcode buffered reader" and "barcodes". Maybe we can remove it. Is there any code in `makeBarcodeBufferedReader()` that knows specifically about barcodes? Does the word "barcode" reduce your effort to understand the code?